src/ng/plugins/groupbar/groupbar.js

```diff
@@ -6,7 +6,7 @@ class Groupbar extends PluginComponent('qgrid.groupbar.tpl.html') {
 	constructor() {
 		super(...arguments);
 
-		this.newGroup = null;
+		this.newSort = null;
```

Comment:
Revert name from newSort to newGroup (for src/ng/plugins/groupbar/groupbar.js file)